app/controllers/concerns/publishable.rb

```diff
@@ -0,0 +1,35 @@
+module Publishable
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :ensure_reviewer, only: [:create, :update, :multiple_update]
+  end
+
+  private
+
+  def ensure_reviewer
```

Comment:
I think this method is doing too much. `ensure_reviewer` makes sense for up to L19 but I think the rest should be its own method